src/lib/__tests__/rateLimiter.test.ts

```diff
@@ -8,15 +8,15 @@ describe("rateLimiter", () => {
         headers: {},
         body: {},
       }
-      expect(skip(req as Request)).toBeFalsy
+      expect(skip(req as Request)).toBeFalsy()
```

Comment:
These tests never ran… 🙈

Reply:
hah, too much ruby in your background dude